clouddriver-openstack/src/main/groovy/com/netflix/spinnaker/clouddriver/openstack/client/OpenstackClientProvider.groovy

```diff
@@ -282,6 +284,42 @@ abstract class OpenstackClientProvider {
   }
 
   /**
+   * Remove load balancer pool.
+   * @param region
```

Comment:
While we're normally not great a generating code docs, it feels strange to see these `param` annotations with descriptions of the variables being passed in.